src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+    <cluster code="0x0752"/>
+    <item name="PerGroupID" value="0x00"/>
+    <item name="AllNodes" value="0x01"/>
+  </enum>
+
+  <struct name="DatastoreGroupKeySetStruct">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="GroupKeySetID" type="int16u"/>
+    <item fieldId="1" name="GroupKeySecurityPolicy" type="DatastoreGroupKeySecurityPolicyEnum" isFabricSensitive="true" min="0x00" max="0x01"/>
+    <item fieldId="2" name="EpochKey0" type="octet_string" length="16" isNullable="true" isFabricSensitive="true" minLength="16"/>
+    <item fieldId="3" name="EpochStartTime0" type="epoch_us" isNullable="true" isFabricSensitive="true"/>
+    <item fieldId="4" name="EpochKey1" type="octet_string" length="16" isNullable="true" isFabricSensitive="true" minLength="16"/>
+    <item fieldId="5" name="EpochStartTime1" type="epoch_us" isNullable="true" isFabricSensitive="true"/>
+    <item fieldId="6" name="EpochKey2" type="octet_string" length="16" isNullable="true" isFabricSensitive="true" minLength="16"/>
+    <item fieldId="7" name="EpochStartTime2" type="epoch_us" isNullable="true" isFabricSensitive="true"/>
+    <item fieldId="8" name="GroupKeyMulticastPolicy" type="DatastoreGroupKeyMulticastPolicyEnum" isFabricSensitive="true" min="0x00" max="0x01"/>
+  </struct>
+
+  <struct name="DatastoreStatusStruct" apiMaturity="provisional" isFabricScoped="true">
+    <cluster code="0x0752"/>
+    <item fieldId="0" name="State" type="DatastoreStateEnum" default="0x00" min="0x00" max="0x03"/>
+    <item fieldId="1" name="UpdateTimestamp" type="epoch_s"/>
+    <item fieldId="2" name="FailureCode" type="status" default="0"/>
+  </struct>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Datastore</name>
+    <code>0x0752</code>
+    <define>JOINT_FABRIC_DATASTORE_CLUSTER</define>
+    <description>Joint Fabric Datastore Cluster.</description>
+
+    <attribute side="server" code="0x0000" name="AnchorRootCA" define="ANCHOR_ROOT_CA" type="octet_string">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0001" name="AnchorNodeID" define="ANCHOR_NODE_ID" type="node_id">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0002" name="AnchorVendorID" define="ANCHOR_VENDOR_ID" type="vendor_id">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0003" name="FriendlyName" define="FRIENDLY_NAME" type="char_string" length="32">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0004" name="GroupKeySetList" define="GROUP_KEY_SET_LIST" type="array" entryType="DatastoreGroupKeySetStruct">
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x0005" name="GroupList" define="GROUP_LIST" type="array" entryType="DatastoreGroupInformationEntryStruct">
```

Comment:
OK, so using "S" on the attribute here is _allowed_, but is the behavior that results the intended one?  As in, it will elide other-fabric entries from the list entirely?

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
The spec change will be tracked using CHIP-Specifications/connectedhomeip-spec#11622. Once the spec update is complete, an SDK PR can be submitted to align it with the updated specification.